Content/0.3.3 - Account History/accountHistory.json

```diff
@@ -0,0 +1,34 @@
+{
+	"accountHistory_title": "History",
+    "accountHistory_transactionClassGeneral": "General",
+    "accountHistory_transactionClassTransfer": "Transfer",
+    "accountHistory_transactionClassStaking": "Staking",
+    "accountHistory_transactionClassUnstaking": "Unstaking",
+    "accountHistory_transactionClassClaim": "Claim",
+    "accountHistory_transactionClassContribute": "Contribute",
+    "accountHistory_transactionClassRedeem": "Redeem",
+    "accountHistory_transactionClassAccountSettings": "Deposit Settings",
+    "accountHistory_transactionClassOther": "Other",
+    "accountHistory_filters_title": "Filter",
+    "accountHistory_filters_clearAll": "Clear All",
+    "accountHistory_filters_showResultsButton": "Show Results",
+    "accountHistory_filters_assetTypeLabel": "Type of Asset",
+    "accountHistory_filters_assetTypeTokenShowAll": "+ Show All Tokens",
+    "accountHistory_filters_assetTypeTokenShowLess": "- Show Less Tokens",
+    "accountHistory_filters_assetTypeNFTShowAll": "+ Show All NFTs",
+    "accountHistory_filters_assetTypeNFTShowLess": "- Show Less NFTs",
+    "accountHistory_filters_assetTypeTokensLabel": "Tokens",
+    "accountHistory_filters_depositsType": "Deposits",
+    "accountHistory_filters_withdrawalsType": "Withdrawas",
+    "accountHistory_filters_assetTypeNFTsLabel": "NFTs",
+    "accountHistory_filters_transactionTypeLabel": "Type of Transaction",
+    "accountHistory_settingsSection": "Settings",
+    "accountHistory_settings_info": "Updated Account Deposit Settings",
+    "accountHistory_noBalanceChanges": "No deposits or withdrawals from this account in this transaction.",
+    "accountHistory_withdrawnSection": "Withdrawn",
+    "accountHistory_depositedSection": "Deposited",
+    "accountHistory_unknownTransactionInfo": "This transaction cannot be summarized. Only the raw transaction manifest may be viewed.",
+    "accountHistory_failedTransaction": "Failed Transaction",
+    "accountHistory_empty": "You have no\nTransactions.",
+    "accountHistory_bullet": "\u2022"
```

Comment:
Will we use different bullets in other languages?

Reply:
I doubt, I will remove it and will be up to wallet to add the bullet